### ISSUE TITLE
Code Insights: Change org control in the dashboard creation UI page

### ISSUE THREAD

**Issue by vovakulikov** [sourcegraph/sourcegraph#22675]
### Background

Originally in the first version of dashboard creation UI, we came up with a simple radio button group list for organization visibility of the dashboard (see screenshot below)

<img width="235" alt="Screenshot 2021-07-08 at 14 23 43" src="https://user-images.githubusercontent.com/18492575/124913698-20328f80-dff8-11eb-9e95-026397fade39.png">

According to the new designs we want to slightly change this representation of orgs and use select org for picking the right org for the dashboard visibility (see next screenshot)

<img width="437" alt="Screenshot 2021-07-08 at 14 24 47" src="https://user-images.githubusercontent.com/18492575/124913940-6b4ca280-dff8-11eb-864a-ca843cfdf480.png">

**Comment by felixfbecker** [comment]
@vovakulikov is this issue still accurate?

**Comment by vovakulikov** [comment]
@felixfbecker Yes. I think this is a part of our next dashboard interaction. I can see this select in our designs https://www.figma.com/file/g5lSlPBnHt00wCogjRFnt5/RFC-398%3A-Easily-Viewing-and-Sharing-Code-Insights-WIP?node-id=1323%3A9347
